File: Mathematical/9. Palindrome Number/Solution.java
class Solution {
    public boolean isPalindrome(int x) {
        String numeroString = "" + x;
        boolean ePalindromo = true;
        int tam = numeroString.length();
        for (int i = 0; i < tam; i++) {
            if (numeroString.charAt(i) != numeroString.charAt(tam - 1 - i)) {
                ePalindromo = false;
                i = tam;
            }
        }
        return ePalindromo;
    }
}

File: Mathematical/9. Palindrome Number/Solution.cpp
class Solution {
public:
    bool isPalindrome(int x) {
       long int pal=0;
        int r,n[10],i=0,cnt=0;
        r=x;
        while(r!=0)
        {
          n[i]=r%10;
          r=r/10;
          i++;
          cnt++;
        }
        for(i=0;i<cnt;i++)
        {
          pal=pal*10;
          pal=pal+n[i];
        }
   if((x>=0)&&(x==pal))
   return 1;
     return 0;
    }
};
